### PR TITLE
Pretasks

### PR DESCRIPTION
**Pull request by sac**

### CODE REVIEW

File: gluster-ansible.spec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 %global rolesdir %{_sysconfdir}/ansible/roles/gluster.ansible
 %global docdir %{_datadir}/doc/gluster.ansible
-%global buildnum 2
+%global buildnum 5
 
-Name:      gluster-ansible-roles
-Version:   1.0.0
-Release:   2%{?dist}
+Name:      gluster-ansible
```
Comment (sabose) [inline gluster-ansible.spec]: why is this part of pre-tasks change?

File: playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml [hunk 2]
```diff
@@ -7,6 +7,44 @@
...
+      failed_when: result.stdout_lines|length == 0 and result.rc != 0
+      when: gluster_features_fqdn_check | default(true)
+      delegate_to: localhost
+      run_once: true
+
+    # Check if /var has enough disk. We check for 15G minimum.
+    - name: Check if /var/log has enough disk space
+      shell: df -m /var/log | awk '/[0-9]%/ {print $4}'
+      register: disk_size
+      failed_when: disk_size.stdout|int < gluster_features_min_disk|default(15360)
```
Comment (sabose) [inline playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml]: are you changing the GiB to GB check here?

File: playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml [hunk 3]
```diff
@@ -7,6 +7,44 @@
...
+    # gluster_features_512B_check to false. DELETE the below task once
+    # OVirt limitation is fixed
+    - name: Check if disks have logical block size of 512B else fail
+      command: cat /sys/block/{{item.pvname|basename}}/queue/logical_block_size
+      register: logical_blk_size
+      failed_when: logical_blk_size.stdout|int != 512
+      when: gluster_infra_volume_groups is defined and
+            item.pvname is not search("/dev/mapper") and
+            gluster_features_512B_check|default(true)
+      with_items: "{{ gluster_infra_volume_groups }}"
```
Comment (sabose) [inline playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml]: can you also add assert statements to the task failures?

File: gluster-ansible.spec (hunk 1 above)
Comment (sac) [reply gluster-ansible.spec]: This was a earlier commit which was done to do a build. And when the PR is created it included earlier commits as well.

File: playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml (hunk 2 above)
Comment (sac) [reply playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml]: Yes. I have done that.

File: playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml (hunk 3 above)
Comment (sac) [reply playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml]: Done.

**Pull request merged**